src/components/atoms/NavigationLogo.vue

```diff
@@ -1,15 +1,13 @@
 <template>
-	<p class="navigation__logo">{{text}}</p>
+	<p class="navigation__logo">{{logo}}</p>
```

Comment:
`logo` variable is text or html string with `img` tag? last variable name was more clear